crates/bw/src/main.rs

```diff
@@ -192,10 +193,11 @@ async fn process_commands() -> Result<()> {
 
             let email = text_prompt_when_none("Email", email)?;
             let password = Password::new("Password").prompt()?;
+            let password = SensitiveString::new(Box::new(password));
```

Comment:
Should we inline password::new to prevent possibly leaking?

Reply:
Done, we'll need to add tests to make sure that the library isn't leaking anything by itself. Not sure how to properly test that yet. Thankfully this isn't used anywhere other than the `bw` crate.